### PR TITLE
Update supported-platforms.md

### LINKED ISSUE #26140: More details info regarding Azure App Service is available but not linked to
Hello there,

There is more up to date information regarding gRPC + ASP.NET core + Azure App Service and it's available here:
https://github.com/Azure/app-service-linux-docs/blob/master/HowTo/gRPC/use_gRPC_with_dotnet.md

We should be linking to this information from this article.

Thank you
Angelos

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 76d3f0ff-f65f-97d2-940f-4509b0d425bd
* Version Independent ID: ea62de0b-d7e5-e543-b892-a587d6e9dc84
* Content: [gRPC on .NET supported platforms](https://docs.microsoft.com/en-us/aspnet/core/grpc/supported-platforms?view=aspnetcore-6.0)
* Content Source: [aspnetcore/grpc/supported-platforms.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/grpc/supported-platforms.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-grpc**
* GitHub Login: @JamesNK
* Microsoft Alias: **jamesnk**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #26140
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/grpc/supported-platforms?view=aspnetcore-3.0&branch=pr-en-us-26142#azure-services)

### CODE REVIEW

File: aspnetcore/grpc/supported-platforms.md [hunk 1]
```diff
@@ -69,6 +69,8 @@ For information about configuring ASP.NET Core servers to run gRPC, see <xref:gr
 
 &dagger;Azure App Service doesn't support hosting gRPC over HTTP/2. gRPC-Web is a compatible alternative.
 
+See [How-to deploy a .NET 6 gRPC app on App Service](https://github.com/Azure/app-service-linux-docs/blob/master/HowTo/gRPC/use_gRPC_with_dotnet.md) for Azure App Service deployment information.
+
 Work is in-progress to improve support for gRPC with HTTP/2 in Azure App Service. For more information, see [this GitHub issue](https://github.com/dotnet/AspNetCore/issues/9020).
```
Comment (Rick-Anderson) [inline aspnetcore/grpc/supported-platforms.md:74]: Can we remove this?

File: aspnetcore/grpc/supported-platforms.md [hunk 2]
```diff
@@ -67,7 +67,9 @@ For information about configuring ASP.NET Core servers to run gRPC, see <xref:gr
 > * [Azure Container Apps](https://azure.microsoft.com/services/container-apps/)
 > * [Azure App Service](https://azure.microsoft.com/services/app-service/)&dagger;
 
-&dagger;Azure App Service doesn't support hosting gRPC over HTTP/2. gRPC-Web is a compatible alternative.
+&dagger;Azure App Service doesn't support hosting gRPC over HTTP/2 everywhere. Support is currently experimental. gRPC-Web is a compatible alternative that's available everywhere.
+
+See [How-to deploy a .NET 6 gRPC app on App Service](https://github.com/Azure/app-service-linux-docs/blob/master/HowTo/gRPC/use_gRPC_with_dotnet.md) for Azure App Service deployment information.
```
Comment (JamesNK) [inline aspnetcore/grpc/supported-platforms.md:72]:
```suggestion
See [How-to deploy a .NET 6 gRPC app on App Service](https://github.com/Azure/app-service-linux-docs/blob/master/HowTo/gRPC/use_gRPC_with_dotnet.md) for Azure App Service deployment information. This page discusses deploying gRPC that uses HTTP/2 to App Service which is experimental and not available in all Azure data centers.
```

File: aspnetcore/grpc/supported-platforms.md (hunk 1 above)
Comment (JamesNK) [reply aspnetcore/grpc/supported-platforms.md:74]: This should be kept. gRPC + Azure + HTTP/2 support is experimental. The GitHub issue will be updated with the latest information.

When gRPC is fully supported on App Service then the link can be removed.

**Comment by JamesNK** [comment]
The linked doc - https://github.com/Azure/app-service-linux-docs/blob/master/HowTo/gRPC/use_gRPC_with_dotnet.md - looks like it's intended to be published. It has `[!WARNING]` syntax in it.

Do you know where the published doc lives? It would be nicer to link to it.

**Comment by Rick-Anderson** [comment]
> The linked doc - https://github.com/Azure/app-service-linux-docs/blob/master/HowTo/gRPC/use_gRPC_with_dotnet.md - looks like it's intended to be published. It has `[!WARNING]` syntax in it.
> 
> Do you know where the published doc lives? It would be nicer to link to it.

My guess is this is not built and published to https://docs.microsoft.com/azure/app-service/ but is just a ReadMe file in MD format. https://github.com/Azure/app-service-linux-docs has no information either way.
cc @btardif @AngelosP who can answer this question.

**Review by JamesNK — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@btardif @AngelosP can you verify if  https://docs.microsoft.com/azure/app-service/ but is just a ReadMe file and not published as a doc?

**Comment by btardif** [comment]
@jeffwmartinez we should make sure we update this doc once GRPC is fully rolled out on App Service.

**Comment by btardif** [comment]
@Rick-Anderson you guys have the right link --> https://github.com/Azure/app-service-linux-docs/blob/master/HowTo/gRPC/use_gRPC_with_dotnet.md there is no official gRPC doc but we are working on it.

**Pull request merged**